agent/agent/config.py

```diff
@@ -0,0 +1,173 @@
+"""Submission core configuration parameters."""
+
+import os
+from kombu.serialization import register
+from .serializer import dumps, loads
+
+register('process-json', dumps, loads,
+         content_type='application/x-process-json',
+         content_encoding='utf-8')
+
+APPLY_RULES = bool(int(os.environ.get('APPLY_RULES', '1')))
+BROKER_URL = os.environ.get('SUBMISSION_AGENT_BROKER_URL',
+                            'redis://localhost:6379/0')
+RESULT_BACKEND = os.environ.get('SUBMISSION_AGENT_RESULT_BACKEND', BROKER_URL)
+QUEUE_NAME_PREFIX = os.environ.get('SUBMISSION_AGENT_QUEUE_NAME_PREFIX',
+                                   'submission-agent-')
+PREFETCH_MULTIPLIER = int(os.environ.get(
+    'SUBMISSION_AGENT_WORKER_PREFETCH_MULTIPLIER',
+    '1'
+))
+TASK_ACKS_LATE = bool(int(os.environ.get('SUBMISSION_AGENT_TASK_ACKS_LATE', '1')))
+
+CELERY_ACCEPT_CONTENT = ['process-json']
+CELERY_TASK_SERIALIZER = 'process-json'
+CELERY_RESULT_SERIALIZER = 'process-json'
+
+FILE_MANAGER_HOST = os.environ.get('FILE_MANAGER_HOST', 'arxiv.org')
+FILE_MANAGER_PORT = os.environ.get('FILE_MANAGER_PORT', '443')
+FILE_MANAGER_PROTO = os.environ.get('FILE_MANAGER_PROTO', 'https')
+FILE_MANAGER_PATH = os.environ.get('FILE_MANAGER_PATH', '')
+FILE_MANAGER_ENDPOINT = os.environ.get(
+    'FILE_MANAGER_ENDPOINT',
+    f'{FILE_MANAGER_PROTO}://{FILE_MANAGER_HOST}:{FILE_MANAGER_PORT}/{FILE_MANAGER_PATH}'
+)
+FILE_MANAGER_VERIFY = bool(int(os.environ.get('FILE_MANAGER_VERIFY', '1')))
+
+COMPILER_HOST = os.environ.get('COMPILER_HOST', 'arxiv.org')
+COMPILER_PORT = os.environ.get('COMPILER_PORT', '443')
+COMPILER_PROTO = os.environ.get('COMPILER_PROTO', 'https')
+COMPILER_PATH = os.environ.get('COMPILER_PATH', '')
+COMPILER_ENDPOINT = os.environ.get(
+    'COMPILER_ENDPOINT',
+    f'{COMPILER_PROTO}://{COMPILER_HOST}:{COMPILER_PORT}/{COMPILER_PATH}'
+)
+COMPILER_VERIFY = bool(int(os.environ.get('COMPILER_VERIFY', '1')))
+
+CLASSIFIER_ENDPOINT = os.environ.get('CLASSIFIER_ENDPOINT', 'http://localhost:8000')
+CLASSIFIER_VERIFY = bool(int(os.environ.get('CLASSIFIER_VERIFY', '0')))
+
+ENABLE_ASYNC = os.environ.get('ENABLE_ASYNC', '0')
```

Comment:
Any reason `ENABLE_ASYNC` and `ENABLE_CALLBACKS` couldn't also be `bool`s?

Reply:
These are actually no longer used; deleted in cbcf088